### PR TITLE
Fix for 500 error on progress page

### PR DESCRIPTION
**Pull request by dyudyunov**
Add management command to generate persistent grade

**Youtrack:**
- https://youtrack.raccoongang.com/issue/FLS-557

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: lms/djangoapps/grades/management/commands/generate_persistent_grade.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+"""
+Management command which update/generate persistent grades
+
+source ~/edxapp_env ; ./manage.py lms generate_persistent_grade --settings=production
```
Comment (GlugovGrGlib) [inline lms/djangoapps/grades/management/commands/generate_persistent_grade.py:4]: Add description what does it mean.
e.g. Usage:
Can be used like this:
or smth similar

Comment (dyudyunov) [reply lms/djangoapps/grades/management/commands/generate_persistent_grade.py]: done

File: lms/djangoapps/grades/management/commands/generate_persistent_grade.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            except:
+                log.warning(
+                    'Persistent grade not generated for the user {} on the course {}'
+                    .format(user.username, enrollment.course_id)
+                )
+            if course_structure:
+                try:
+                    CourseGradeFactory().update(user, course, course_structure, force_update_subsections=True)
+                except:
+                    pass
```
Comment (GlugovGrGlib) [inline lms/djangoapps/grades/management/commands/generate_persistent_grade.py:43]: Please log an error

File: lms/djangoapps/grades/management/commands/generate_persistent_grade.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    help = "Update/generate persistent grades"
+
+    def handle(self, *args, **options):
+        course_structure = None
+        for enrollment in CourseEnrollment.objects.all():
+            user = enrollment.user
+            try:
+                course = courses.get_course_by_id(enrollment.course_id)
+                course_structure = get_course_blocks(user, course.location)
+            except:
```
Comment (GlugovGrGlib) [inline lms/djangoapps/grades/management/commands/generate_persistent_grade.py:34]: Please log an actual error text

File: lms/djangoapps/grades/management/commands/generate_persistent_grade.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+class Command(BaseCommand):
+    """
+    Management command which update/generate persistent grades.
+    """
+
+    help = "Update/generate persistent grades"
+
+    def handle(self, *args, **options):
+        course_structure = None
+        for enrollment in CourseEnrollment.objects.all():
```
Comment (GlugovGrGlib) [inline lms/djangoapps/grades/management/commands/generate_persistent_grade.py:29]: Please use djnago iterator here

Comment (GlugovGrGlib) [reply lms/djangoapps/grades/management/commands/generate_persistent_grade.py:29]: https://docs.djangoproject.com/en/1.11/ref/models/querysets/#iterator

**Review by GlugovGrGlib — CHANGES_REQUESTED** [review]

File: lms/djangoapps/grades/management/commands/generate_persistent_grade.py (hunk 2 above)
Comment (dyudyunov) [reply lms/djangoapps/grades/management/commands/generate_persistent_grade.py]: done

File: lms/djangoapps/grades/management/commands/generate_persistent_grade.py (hunk 3 above)
Comment (dyudyunov) [reply lms/djangoapps/grades/management/commands/generate_persistent_grade.py]: done

File: lms/djangoapps/grades/management/commands/generate_persistent_grade.py (hunk 4 above)
Comment (dyudyunov) [reply lms/djangoapps/grades/management/commands/generate_persistent_grade.py]: done

**Review by OPersian — APPROVED** [review]

**Review by GlugovGrGlib — APPROVED** [review]

**Pull request merged**